### ISSUE TITLE
Won't Do Manual Backups

### ISSUE THREAD

**Issue by GoogleCodeExporter** [asm0dey/flyback#49]
```
What steps will reproduce the problem?
1. Click the clock icon
2.
3.

What is the expected output? What do you see instead?
Backup. This message:
External Storage Location Error

The external storage location you've specified is already in use.  Please
quit any other open instances of FlyBack (or wait for their backups to
complete) before starting a new backup.

What version of the product are you using? On what operating system?
0.4.0 Xubuntu Hardy Linux

Please provide any additional information below.
I've been able to do manual backups in the past, but never timed. The time
problem may be because I want them done on Sunday, and I don't know if
Sunday is 0, 1, 6, or 7.

I've not changed the location since I was able to do the manuals. It's my
Windows partition, and is not used for anything else in Linux.

Lane

```

Original issue reported on code.google.com by `lane.lester@gmail.com` on 2 Apr 2008 at 11:53
